samcli/lib/build/workflow_config.py

```diff
@@ -116,31 +121,28 @@ def get_selector(selector_list, identifiers, specified_workflow=None):
     """
 
     # Create a combined view of all the selectors
-    all_selectors = {}
+    all_selectors: Dict[str, Union["BasicWorkflowSelector", "ManifestWorkflowSelector"]] = dict()
```

Comment:
Would it make sense to use type aliases for repeated types? e.g. `Union["BasicWorkflowSelector", "ManifestWorkflowSelector"]`?